CHANGES.rst

```diff
@@ -22,6 +22,8 @@ ramp_fitting
 - Updating how NaNs and DO_NOT_USE flags are handled in the rateints
   product. [#112]
 
+- Updating how GLS handles bad gain values. [#115]
```

Comment:
Could include a few more details here on what the new handling is, such as the fact that pixels with bad gain values are now flagged as bad and set to zero.

Reply:
I expanded this to more fully flesh it out.